lms/static/js/edxnotes/plugins/ai_summarize.js

```diff
@@ -0,0 +1,101 @@
+(function(define, undefined) {
+    'use strict';
+    define(['jquery', 'underscore', 'annotator_1.2.9'], function($, _, Annotator) {
+    /**
+     * AiSummarize plugin adds a button to the annotatorjs adder in order to
+     * summarize the text selected and save as annotation.
+     **/
+        Annotator.Plugin.AiSummarize = function() {
+            Annotator.Plugin.apply(this, arguments);
+        };
+
+        $.extend(Annotator.Plugin.AiSummarize.prototype, new Annotator.Plugin(), {
+            pluginInit: function() {
+                // Overrides of annotatorjs HTML/CSS to add summarize button.
+                var style = document.createElement('style');
+                style.innerHTML = `
```

Comment:
Is there a way to define these styles in a SASS or CSS file? @alexjmpb

Reply:
I wanted to keep all the functionality inside the plugin, do you think we should include it in another file?

Reply:
This is not desirable, but as we want to avoid a lot of changes to the code base, I think it is fine in this particular case. @alexjmpb